sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.9.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.9.0 (2025-04-08)
```

Comment:
https://azure.github.io/azure-sdk/policies_releases.html#changelog-entries-for-stable-releases

Reply:
Updated